src/main/scala/com/gu/zuora/ZuoraConfig.scala

```diff
@@ -9,7 +9,7 @@ object ZuoraConfig {
   type RatePlanId = String
 }
 
-case class ZuoraContributionConfig(productRatePlanId: RatePlanId, productRatePlanChargeId: RatePlanId)
+case class ZuoraContributionConfig(productRatePlanId: RatePlanId, productRatePlanChargeId: RatePlanId /*really?*/ )
```

Comment:
I don't think this is a RatePlanId (although it can be represented as a String).  Wasn't sure whether to refactor it in this pr though?